robot_ws/src/turtlebot_controller/robomaker/inference_worker.py

```diff
@@ -94,12 +95,16 @@ def load_graph(self):
 
     def callback_image(self, raw_image):
         # Read the image
+        global world_name
         image_data = np.array(raw_image.data)
         image = Image.frombytes('RGB', (raw_image.width, raw_image.height),
                                 image_data, 'raw', 'BGR', 0, 1)
         image = image.resize(TRAINING_IMAGE_SIZE)
         image = np.array(image)
 
+        if world_name=='brick_wall':
```

Comment:
I think the logic here should be to do Gaussian blur based on a launch parameter AND when the world is not `empty`. The default value for the "`do_blur`" launch parameter can be `True`.

Reply:
As per your suggestion, I have added DO_BLUR variable to control filter enable/disable 
f61daf1 . Default value set to true.  Also, we have added a model trained in a brick wall environment for reference b307578.